api/soap/mantisconnect.wsdl

```diff
@@ -546,6 +550,16 @@
   <part name="issue" type="tns:IssueData" /></message>
 <message name="mc_issue_addResponse">
   <part name="return" type="xsd:integer" /></message>
+<message name="mc_issue_reportRequest">
+  <part name="username" type="xsd:string" />
```

Comment:
If we keep the concept of having a user + limited token, then this field would be useful.  If not, then we don't need to get an empty user here just to ignore it internally.